doc/changelog.md

```diff
@@ -4,6 +4,30 @@
 
 <h3>Improvements</h3>
 
+* Catalyst now supports return statements inside conditionals in `@qjit(autograph=True)` compiled
+  functions.
+  [(#)]()
+
+  The following is now possible:
+
+  ```py
+  @qjit(autograph=True)
+  @qml.qnode(qml.device(backend, wires=1))
+  def f(x: float):
+    qml.RY(x, wires=0)
+
+    m = measure(0)
+    if not m:
+        return 0
```

Comment:
Almost contradicted the requirements in my example 😅 Unfortunate that you can't (easily) sub in constants for a measurement process though:
```suggestion
    if not m:
        return qml.expval(qml.PauliZ(0))
```